### DISCUSSION TITLE
Question: how do a view logs when not using vscode?

### DISCUSSION THREAD

**Discussion by tanj** [microsoft/pyright#5052] [General]
I'm trying to troubleshoot a lsp config issue in the helix editor and I'd like to view the pyright logs, but I can't find where the logs are actually output to. The docs only talk about looking at the output in VScode, but that doesn't make sense for this use case. Is there a way to attach to a running instance to capture the extra logging?

The issue I'm trying to debug is why the helix workspace config doesn't seem to have any effect on the pyright `venv` or `venvPath` settings. If I set that config in the `pyrightconfig.json` file everything works fine, but if I have it set in the helix workspace config it doesn't.

System: Windows 10 Pro
pyright version 1.1.305

**Comment by erictraut** [comment]
Converting to a discussion topic since this isn't a bug report.

You are probably better off asking this question in a forum about the Helix editor or its LSP support.

**Reply by tanj** [reply]
My question isn't about helix, but about how `pyright` handles logs. I saw from the pyright docs that there is a verbose flag, but they only mention vscode output tab as a place to read them.

Maybe I'm missing a section in the docs that would explain what I'm after. Right now I'm feeling like I'll need to try and grok the pyright code base to figure this out. (Which I'll do if I have to, but was not really were I wanted to spend my time when figuring out my own issue)

1. Where do the `--verbose` messages go?
2. Is it possible to attach a debugger to a running `pyright` language server? (One launched by an editor other than VSCode)
